src/transforms/reduce/transform.rs

```diff
@@ -147,7 +193,17 @@ impl Reduce {
             flush_period: config.flush_period_ms,
             end_every_period: config.end_every_period_ms,
             group_by,
-            merge_strategies: config.merge_strategies.clone(),
+            merge_strategies: config
+                .merge_strategies
+                .iter()
+                .filter_map(|(path, strategy)| {
+                    // TODO Invalid paths are ignored to preserve backwards compatibility.
+                    //      Merge strategy paths should ideally be [`lookup_v2::ConfigTargetPath`]
+                    //      which means an invalid path would result in an configuration error.
+                    let parsed_path = parse_target_path(path).ok();
```

Comment:
Am I correct in understanding that previously if an invalid path was given to `merge_strategies` that it would be silently ignored later? If that's true, this change seems fine though I might suggest we take the opportunity to add a warning message.

We could also choose to start bubbling up these errors now so that users get them at start-time.

Reply:
Correct, added a warning.